README.md

```diff
@@ -29,6 +29,8 @@ After having installed the latest Zivid SDK, the easiest way to install Zivid Py
 
     pip install zivid
 
+Note! If you don't use the latest Zivid SDK version you need to manually specify the version. See [Versioning](#versioning).
```

Comment:
Not a biggie, but when we convert .rst to .md for notes we use:

Note:

```
> Support for Ubuntu 16.04 is removed from SDK 2.7.

-----
```
See https://github.com/zivid/zivid-cpp-samples